packages/node-core/src/indexer/storeCache/storeCache.service.ts

```diff
@@ -51,6 +55,13 @@ export class StoreCacheService extends BaseCacheService {
     this.poiRepo = poi;
   }
 
+  get unfinalizedBlocks(): boolean {
+    return this._unfinalizedBlocks;
+  }
+  get indexCountLimit(): number {
+    return this._indexCountLimit;
+  }
```

Comment:
These don't belong here, they don't relate specificly to store cache and are just a proxy for node config